### PR TITLE
Olofk-awslinux

### PR DESCRIPTION
**Pull request by olofkallander**
Build config for AWS-linux2

### CODE REVIEW

File: docker/aws-linux/runtests.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    return 1
+  fi
+
+  # Convert coverage data from LLVM to lcov format, see http://logan.tw/posts/2015/04/28/check-code-coverage-with-clang-and-lcov/
+  # (With a more recent LLVM, the same can be achieved with `llvm-cov export lcov`)
+  echo '#!/bin/bash' > llvm-gcov.sh
+  echo 'llvm-cov gcov "$@"' >> llvm-gcov.sh
+  chmod +x llvm-gcov.sh
+
+  lcov --capture \
```
Comment (RicardoMDomingues) [inline docker/aws-linux/runtests.sh:28]: I didn't find `lcov` installation on `dockerfile`. Is it included in other package that is being installed?

**Comment by olofkallander** [comment]
> I find away to install clang without build it. We can download the release form github and copy the artefacts to `/usr/local/*`. Here is how we can do it. 

```
RUN cd /tmp \
    && wget -O clang+llvm-12.0.1-x86_64-linux.tar.xz https://github.com/llvm/llvm-project/releases/download/llvmorg-12.0.1/clang+llvm-12.0.1-x86_64-linux-gnu-ubuntu-16.04.tar.xz \
    && tar xvf clang+llvm-12.0.1-x86_64-linux.tar.xz \
    && cd clang+llvm-12.0.1-x86_64-linux-gnu-ubuntu-* \
    && cp -rn * /usr/local \
    && cd /tmp && rm -rf clang+llvm-12.0.1-x86_64-linux-gnu-ubuntu-*
``` 

ok, I will try that. Should we go for clang 12 on el8 and ubuntu also then?

**Comment by olofkallander** [comment]
> I didn't find `lcov` installation on `dockerfile`. Is it included in other package that is being installed? 

Good. I will remove it from runtests. I assume it is sufficient to run lcov on some platform not all.

**Comment by RicardoMDomingues** [comment]
> > I find away to install clang without build it. We can download the release form github and copy the artefacts to `/usr/local/*`. Here is how we can do it.
> 
> ```
> RUN cd /tmp \
>     && wget -O clang+llvm-12.0.1-x86_64-linux.tar.xz https://github.com/llvm/llvm-project/releases/download/llvmorg-12.0.1/clang+llvm-12.0.1-x86_64-linux-gnu-ubuntu-16.04.tar.xz \
>     && tar xvf clang+llvm-12.0.1-x86_64-linux.tar.xz \
>     && cd clang+llvm-12.0.1-x86_64-linux-gnu-ubuntu-* \
>     && cp -rn * /usr/local \
>     && cd /tmp && rm -rf clang+llvm-12.0.1-x86_64-linux-gnu-ubuntu-*
> ```
> 
> ok, I will try that. Should we go for clang 12 on el8 and ubuntu also then?

I don't see any reason to not do it. I think we should try to keep our tools updated

**Review by RicardoMDomingues — APPROVED** [review]

File: Jenkins/PRUnitTestRunner.groovy [hunk 2]
```diff
@@ -19,23 +19,23 @@ abortPreviousRunningBuilds()
 
 parallel "Release": {
     node('be-integration') {
-        prRunner("Release", "el7")
+        prRunner("Release", "ubuntu-focal-deb")
```
Comment (RicardoMDomingues) [inline Jenkins/PRUnitTestRunner.groovy:22]: Duplicated of line 54

**Pull request merged**